ui/raidboss/data/06-ew/ultimate/the_omega_protocol.ts

```diff
@@ -8,6 +8,16 @@ import { PluginCombatantState } from '../../../../../types/event';
 import { NetMatches } from '../../../../../types/net_matches';
 import { TriggerSet } from '../../../../../types/trigger';
 
+type Phase =
+  | 'omega'
+  | 'm/f'
+  | 'final-omega'
+  | 'blue-screen'
+  | 'delta-version'
+  | 'sigma-version'
+  | 'omega-version'
+  | 'alpha-omega';
```

Comment:
Sorry for nitpicking, but these are all pretty wordy.  What about p1, p2, p3, p4, delta, sigma, omega, p6? These correspond to what fflogs (and everybody else I know uses) but still splits out p5 (which I think is a good idea).

Reply:
No worries. It was complicated thinking up a name for it and seeing already what we placed in the timeline, yet another difference in phase names.